pybsn/switch.py

```diff
@@ -0,0 +1,20 @@
+class Switch(object): 
+
+    def __init__(self):
```

Comment:
I think it would make sense to have a private constructor for the REST responses. And have a public one for stuff that is actually configurable. 

Maybe keep a hash of what fields are actually configurable.